src/aparna/scannerClassConcept/CalculationThroughScanner.java

```diff
@@ -0,0 +1,79 @@
+/*Assignment - 14 : 8th April'2021
+Use Scanner Class for this Program
+Write a class called CalculationThroughScanner with following functionality.
+a. add
+b. subtraction
+c. division
+d. multiplication
+e. printTotalAnswer.
+
+Method a,b,c,d should have 2 parameters and return int value.
+printTotalAnswer method should have 4 parameters and doesnt return anything.
+Note : do not create any instance variable.
+ * 
+ */
+package aparna.scannerClassConcept;
+
+import java.util.Scanner;
+
+public class CalculationThroughScanner {
+
+	double addition(double number1, double number2) {
+		double additonResult = number1 + number2;
+		System.out.println("Addition  of " + number1 + " and  " + number2 + "  is : " + additonResult);
+		return additonResult;
+	}
+
+	double subtraction(double number1, double number2) {
+		double subtractionResult = number1 - number2;
+		System.out.println("Subtraction of " + number1 + " and  " + number2 + "  is : " + subtractionResult);
+		return subtractionResult;
+	}
+
+	double multiplication(double number1, double number2) {
+		double multiplicationResult = 0;
```

Comment:
for multiplication this not require any way if you are multiplying with 0 then answer will be 0.